CONTRIBUTING.MD

```diff
@@ -0,0 +1,101 @@
+# Contributing to `dbt-snowflake`
+
+1. [About this document](#about-this-document)
+3. [Getting the code](#getting-the-code)
+5. [Running `dbt-snowflake` in development](#running-dbt-snowflake-in-development)
+6. [Testing](#testing)
+7. [Updating Docs](#updating-docs)
+7. [Submitting a Pull Request](#submitting-a-pull-request)
+
+## About this document
+This document is a guide intended for folks interested in contributing to `dbt-snowflake`. Below, we document the process by which members of the community should create issues and submit pull requests (PRs) in this repository. It is not intended as a guide for using `dbt-snowflake`, and it assumes a certain level of familiarity with Python concepts such as virtualenvs, `pip`, python modules, filesystems, and so on. This guide assumes you are using macOS or Linux and are comfortable with the command line.
```

Comment:
Could we actually point people to the dbt docs for using dbt-snowflake maybe :) 

Also, one thing I fail at but try to work at is writing shorter sentences. Let's break up the long one in here!